### PR TITLE
fix(file upload): change error text of uploaded file from byte to mb

### PR DESCRIPTION
**Pull request by Usmanfee**
## Description
converted file size from Bytes to Mega-Byte of uploaded document for better UX.
Screenshot 2024-09-30 at 17 13 39

Changelog entry:
```
- **File Upload**:
  - changed error text of uploaded file from Byte to MB [#866](https://github.com/eclipse-tractusx/portal-frontend/pull/1185)
```

## Why
It is hard for user to understand long digits in Bytes and easier to read in mega byte size for better understanding and usually the size of the file either comes in KB or MB.

## Issue
https://github.com/eclipse-tractusx/portal-frontend/issues/1184

## Checklist

Please delete options that are not relevant.

- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: src/components/shared/basic/Dropzone/index.tsx [hunk 1]
```diff
@@ -111,6 +111,19 @@ export const Dropzone = ({
     [currentFiles, onChange, handleDelete]
   )
```
Comment (shubhamv-ss) [inline src/components/shared/basic/Dropzone/index.tsx:113]:
```suggestion
const handleFileSizeValidator = (file: File) => {
  if (!maxFileSize) return null;
  const fileSizeInMB = (file.size / 1048576).toFixed(2);
  return file.size > maxFileSize
    ? { code: 'size-too-large', message: `File is larger than ${fileSizeInMB} MB` }
    : null;
}

```

Comment (shubhamv-ss) [reply src/components/shared/basic/Dropzone/index.tsx:113]: Also if possible, put this `1048576` in constant.

File: CHANGELOG.md [hunk 2]
```diff
@@ -152,6 +152,9 @@
 - fixed unchanged text of button when user requests subscription [#985](https://github.com/eclipse-tractusx/portal-frontend/pull/985)
 - fixed height for "Admin Service Detail" page content [#1001](https://github.com/eclipse-tractusx/portal-frontend/pull/1001)
 
+- **Display Uplaoded file error text in MB**:
+  - Fixed error text of uplaoded file from Byte to MB [#866](https://github.com/eclipse-tractusx/portal-frontend/issues/1184)
```
Comment (JordanGerada3) [inline CHANGELOG.md:156]: Can you link instead to PR?

**Review by JordanGerada3 — APPROVED** [review]
LGTM!

**Review by shubhamv-ss — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 3]
```diff
@@ -152,6 +152,9 @@
 - fixed unchanged text of button when user requests subscription [#985](https://github.com/eclipse-tractusx/portal-frontend/pull/985)
 - fixed height for "Admin Service Detail" page content [#1001](https://github.com/eclipse-tractusx/portal-frontend/pull/1001)
 
+- **Display Uplaoded file error text in MB**:
+  - Fixed error text of uplaoded file from Byte to MB [#866](https://github.com/eclipse-tractusx/portal-frontend/pull/1185)
```
Comment (evegufy) [inline CHANGELOG.md:156]: could you please move the update of the changelog to the unreleased section? (first rebase to main)

```suggestion
- **File Upload**:
  - changed error text of uploaded file from Byte to MB [#866](https://github.com/eclipse-tractusx/portal-frontend/pull/1185)
```

**Review by evegufy — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (Usmanfee) [reply CHANGELOG.md]: updated.

**Review by evegufy — APPROVED** [review]

File: src/components/shared/basic/Dropzone/index.tsx [hunk 4]
```diff
@@ -111,6 +112,17 @@ export const Dropzone = ({
...
 
+  const handleFileSizeValidator = (file: File) => {
+    if (!maxFileSize) return null
+    const fileSizeInMB = (file.size / CONVERT_TO_MB).toFixed(2)
+    return file.size > maxFileSize
+      ? {
+          code: 'size-too-large',
+          message: `File is larger than ${fileSizeInMB} MB`,
```
Comment (oyo) [inline src/components/shared/basic/Dropzone/index.tsx:121]: translate message with `t('shared.dropzone.error.fileTooLarge')` and add the text in the assets/locale/ files.

**Review by oyo — CHANGES_REQUESTED** [review]
message hard coded in english

**Comment by Usmanfee** [comment]
@oyo  I have addressed the feedback. Can you please review now ? Thanks

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- **File Upload**:
+  - changed error text of uploaded file from Byte to MB [#866](https://github.com/eclipse-tractusx/portal-frontend/pull/1185)
```
Comment (evegufy) [inline CHANGELOG.md:6]: Hi @Usmanfee we have an continuous issue in the changelog file. I suggest you revert any change to the changelog and add the information which should be added to the changelog in the PR description and I add it then at the next release to the changelog file, sorry for the inconvenience.

**Review by evegufy — APPROVED** [review]

**Pull request merged**